examples/first-frame-network/main.cpp

```diff
@@ -61,18 +61,43 @@ int main(int argc, char *argv[]) {
     }
 
     auto camera = cameras.front();
-    status = camera->initialize();
+
+    // user can pass any config.json stored anywhere in HW
+    status = camera->setControl("initialization_config",
+                                "config/config_walden_nxp.json");
     if (status != Status::OK) {
-        LOG(ERROR) << "Could not initialize camera!";
+        LOG(ERROR) << "Could not set the initialization config file!";
         return 0;
     }
 
+    status = camera->setControl("powerUp", "call");
```

Comment:
"powerUp" is not used internally (we haven't come to a conclusion whether we still need to expose this control on not). So there is no reason to be called here.

Reply:
removed